frontend/src/metabase/dashboard/components/Dashboard/DashboardEmptyState/DashboardEmptyState.tsx

```diff
@@ -48,6 +48,7 @@ export function DashboardEmptyState({
                     creationType: "custom_question",
                     collectionId: dashboard.collection_id ?? undefined,
                     cardType: "question",
+                    dashboardId: dashboard.id,
```

Comment:
Assuming we want the same logic here as when we create a new question from the "Add Questions" button